.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,40 @@
+name: GitHub Actions
+
+on:
+  push:
+    branches:
+    - master
+    - develop
+    - release*
+  pull_request:
+    branches:
+    - master
+    - develop
```

Comment:
should `release*` be here as well?

Reply:
This should probably be only branches your pull requests are going to target.  If you plan to make pull requests against release branches, then `release*` should be here as well.